packages/extension-browser/src/devtools/views/powered-by.tsx

```diff
@@ -16,6 +17,7 @@ const PoweredBy = ({ className }: Props) => {
             <ExternalLink href="https://webhint.io">
                 webhint
             </ExternalLink>
+            <Version />
```

Comment:
Maybe this is simple enough to not bother with a separate component. If we move `const version = require('../../manifest.json').version;` to this file we should be able to replace this with the following with no additional classes or styles needed.
```suggestion
            {' '}
            v{version}
```